client/src/Components/Popup/Popup.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+
+const Popup = (props) => {
+	return (
+		<>
+			{
+				props.error ?
+					(
+						<div className='alert alert-danger' role='alert'>{props.message}</div>
+					) :
+					(
+						<div className='alert alert-success' role='alert'>{props.message}</div>
+					)
+			}
+		</>
+	);
+};
```

Comment:
The alternative clearer approach:
```suggestion
const Popup = (props) => {
        return <div className={`alert alert-${props.error ? 'danger' : 'success'}`} role='alert'>{props.message}</div>
};
```